app/models/offer.rb

```diff
@@ -0,0 +1,6 @@
+class Offer < ApplicationRecord
+  validates :description, presence: true, length: { minimum: 5 }
+  validates :title, presence: true
+  validates :product, presence: true
+  validates :suggested_price, numericality: { only_integer: true }
```

Comment:
what about `status`? You could add some validations for status as well.

Reply:
Added